client/search-ui/src/components/ResultContainer.tsx

```diff
@@ -161,7 +161,12 @@ export const ResultContainer: React.FunctionComponent<React.PropsWithChildren<Re
             role="none"
         >
             <div className={styles.header}>
-                <Icon role="img" className="flex-shrink-0" as={icon} aria-hidden={true} />
+                <Icon
+                    role="img"
+                    className="flex-shrink-0"
+                    as={icon}
+                    aria-label={resultType ? `Result type ${resultType}` : undefined}
```

Comment:
nit: reword this label, this sounds more natural to me:
```suggestion
                    aria-label={resultType ? `${resultType} result` : undefined}
```